test/TrieExampleTest.mo

```diff
@@ -34,5 +34,5 @@ debug {
   func equalKV(a : (Text, Nat), b : (Text, Nat)) : Bool { a == b };
   assert (isSubSet(actual, expected, equalKV));
   assert (isSubSet(expected, actual, equalKV));
-
+  assert (Trie.isValid(t2, true));
```

Comment:
```suggestion
  assert Trie.isValid(t2, true);
```
Parentheses are redundant after `assert`.

Reply:
Oops.  I meant to commit this suggestion, but merged to quickly.

Will fix in a subsequent PR.  This set of "unit tests" needs more attention in the near future, I think.

I'll plan to extend it this week.

Reply:
Aside: More thoughts about the extra parens (cc @rvanasa):
- Seems like the lines above could use the same simplification too.  I was imitating them.
- Maybe Motoko could have a syntax linter help with this, more systematically?  (e.g., the VM's AST can witness the extra parens, I believe.  Maybe the compiler can do that as well.  Either way, VSCode could present that info, eventually.)